package.json

```diff
@@ -22,8 +22,11 @@
     "@types/chai": "4.3.4",
     "@types/mocha": "10.0.1",
     "@types/node": "18.14.0",
+    "@typescript-eslint/eslint-plugin": "^5.56.0",
```

Comment:
What's your opinion on hats (`^`)? I've never liked them as I am not certain what versions I am using, and I have to trust authors to follow semver.

Reply:
Honestly that's why we have package-lock. It's purpose really is exactly this, and also so we can go back later and be really sure we're testing with the version that was checked in.